packages/docs/tutorial/tutorial-requests.yaml

```diff
@@ -20,190 +20,213 @@ _ref:
 
-            Create a file called `.env` in your project directory with the following:
+            ##### `pages/new-ticket.yaml`
 
-            ##### `.env`
-            ```
-            LOWDEFY_SECRET_SHEETS_CLIENT_EMAIL=__YOUR_CLIENT_EMAIL__
-            LOWDEFY_SECRET_SHEETS_PRIVATE_KEY=__YOUR_ENCODED_PRIVATE_KEY__
+            ```yaml
+            id: new-ticket
+                # ...
+            blocks:
+              - id: content_card
+                # ...
+                blocks:
+                  - id: page_heading
+                    type: Title
+                    properties:
+                      content: Log a ticket # Change the title on the page.
+                      level: 3 # Make the title a little smaller (an html `<h3>`).
+                  - id: ticket_title
+                    type: TextInput
+                    properties:
+                      title: Title
+                  - id: ticket_type
+                    type: ButtonSelector
+                    properties:
+                      title: Ticket type
+                      options: # Set the allowed options
+                        - Suggestion
+                        - Complaint
+                        - Question
+
+            ################ ------- Copy from here -------- ################
+                  - id: product
+                    type: Selector
+                    required: true
+                    properties:
+                      title: Product
+                      options: # Set the allowed options QUERY FROM API
+                        _array.map:
+                          - _request: get_products.data.products
+                          - _function:
+                              __args: 0.title
+            ################ ------- Copy to here -------- ################
+
+                  - id: ticket_description
+                    type: TextArea
+                    properties:
+                      title: Description
             ```
 
-            Fill in the client email from the credentials file and your encoded private key values (without quotes). This will give the development server access to these secrets.
+            ### What happened
 
-      - id: warning_dont_commit
-        type: Alert
-        properties:
-          type: warning
-          icon: AiFillWarning
-          description: The .env file contains your credentials. Do not commit this file to git. It should be listed in your .gitignore file if you used the CLI init command.
+            We defined the `AxiosHttp` connection to be used to obtain the list of products that tickets can be created for.
+
+            We also defined an `AxiosHttp` request, to fetch the products data when the page is mounted.
```

Comment:
Maybe something like: 
We also defined an `AxiosHttp` request, to fetch the products data when the page is mounted.

We used the `onMount` event to execute the `Request` action to make the API call.

We used the request response int the `product` selector by using the `_request` and `_array.map` operators.

Reply:
Change made in 9654eef9a0e0ece57d5e66f6400d8265e9346466